cli/src/plugins/javadoc/index.ts

```diff
@@ -123,15 +124,37 @@ async function processClassDoc(
       title: md.text(doc.asString),
       makeBody: makeClassDocPageBody,
     })
-  );
+  );    
   project.writePage(new Page(pageUri, root));
+
+  // also write the package because package names aren't readily available in parsed packages...
+  const pkgUri = `/${doc.containingPackage.name}` // '/' + doc.containingPackage.name.replace(/\./g,'/')
+  const pkg = md.root(
+    makeSection({
+      project,
+      pageUri: pkgUri,
+      doc,
+      depth: 1,
+      title: md.text(doc.containingPackage.name),
+      makeBody: (args: MakeSectionArgs) => {return []},
+    })
+  );
+
+  project.declareEntity({
+    canonicalName: pkgUri,
+    pageUri: pkgUri,
+    data: {
+      category: "package",
+    },
+  })
 }
 
 async function processPackageDoc(
   project: Project<JavadocEntityData>,
   doc: ParsedPackageDoc
 ): Promise<void> {
-  // TODO
+  // Nothing here because parsed package docs are not comprehensive. Instead, we
+  // identify packages as we process classes.
```

Comment:
Is this something we need to fix in the javadoc doclet?

Reply:
Maybe? Personally I think it's better to handle packages "on the fly" as we visit classes -- after all, packages don't really contain any useful information of their own in the javadoc anyway. Maybe that's something to handle down the road when we figure out where that missing "landing page" content is?

Reply:
Sounds good.